pkg/agent/variables.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/blang/semver"
 )
 
+var dockerShimFlags = []string{"--image-pull-progress-deadline", "--network-plugin", "--cni-bin-dir", "--cni-conf-dir", "--docker-endpoint", "--network-plugin", "--network-plugin-mtu"}
```

Comment:
Should we also need to add `--cni-cache-dir` in this list?

Reply:
Thanks! I've added it to the list. `--cni-cache-dir` is not present in RP/AB at the moment, so it should be good.